internal/sirius/payment_test.go

```diff
@@ -273,3 +273,74 @@ func TestPaymentByID(t *testing.T) {
 		})
 	}
 }
+
+func TestFeeReductionByID(t *testing.T) {
+	t.Parallel()
+
+	pact := newPact()
+	defer pact.Teardown()
+
+	testCases := []struct {
+		name             string
+		setup            func()
+		cookies          []*http.Cookie
+		expectedError    func(int) error
+		expectedResponse Payment
+	}{
+		{
+			name: "OK for fee reduction",
+			setup: func() {
+				pact.
+					AddInteraction().
+					Given("I have an lpa which has a fee reduction").
+					UponReceiving("A request for that fee reduction by ID").
+					WithRequest(dsl.Request{
+						Method: http.MethodGet,
+						Path:   dsl.String("/lpa-api/v1/payments/124"),
+					}).
+					WillRespondWith(dsl.Response{
+						Status: http.StatusOK,
+						Body: dsl.Like(map[string]interface{}{
+							"id":               dsl.Like(124),
+							"paymentEvidence":  dsl.String("Some fee reduction evidence"),
+							"source":           dsl.Like("FEE_REDUCTION"),
+							"paymentDate":      dsl.String("23/01/2022"),
+							"feeReductionType": dsl.String("REMISSION"),
+							"case": dsl.Like(map[string]interface{}{
+								"id": dsl.Like(801),
+							}),
+						}),
+						Headers: dsl.MapMatcher{"Content-Type": dsl.String("application/json")},
+					})
+			},
+			expectedResponse: Payment{
+				ID:               124,
+				PaymentEvidence:  "Some fee reduction evidence",
+				FeeReductionType: "REMISSION",
+				Source:           "FEE_REDUCTION",
+				PaymentDate:      DateString("2022-01-23"),
+				Case:             &Case{ID: 801},
```

Comment:
The `I have an lpa which has a fee reduction` step returns a case with an id of `802` so might be good to change for consistency